### DISCUSSION TITLE
My BOX version and some help with the first steps

### DISCUSSION THREAD

**Discussion by RafadlCampo** [probonopd/MiniDexed#266] [Q&A]
Hi,

I gonna share with you my box version, from an old router. 

And I would like to ask if someone can help me with the sound after the principal sound, it sounds like an effect, but I can't turn off. Watch the [VIDEO](https://drive.google.com/file/d/1WPVqZMi9IoslI2O_KeiRVQG6g9HJqMuC/view?usp=sharing). Could be a problem with my keyboard?

![20220601_192244](https://user-images.githubusercontent.com/105709655/171509782-95a23024-9a84-4b21-8deb-876bdbc8db8d.jpg)

**Comment by BobanSpasic** [comment]
Did you check if the TG2 to TG8 are listening to the same MIDI channel or are set to OMNI?

**Reply by RafadlCampo** [reply]
They are set to Omni, and TG2 to TG8 in the Voice 32, that is the problem. . . 

Do you know where can I find some examples of use and configuration

Thank you!!

**Reply by BobanSpasic** [reply]
Omni means listening on all MIDI channels. Change the value from Omni to some channel 1-16, take different value for every TG.
Now, if you have only your TG1 on e.g. channel 1, and you set your keyboard to send on channel 1 - just the TG1 will play.

**Reply by probonopd** [reply]
That's a cool housing. Who needs a 3D printer when there are spare routers. The name on it even sounds musical ;-)
